src/vaccel.c

```diff
@@ -60,16 +60,23 @@ static int load_backend_plugin(const char *path)
 
 static int load_backend_plugins(char *plugins)
 {
-	char *plugin = strtok(plugins, ";");
-	while (plugin) {
-		int ret = load_backend_plugin(plugin);
-		if (ret != VACCEL_OK)
-			return ret;
+        char *plugin, *plugins_tmp;
 
-		plugin = strtok(NULL, ";");
-	}
+        plugins_tmp = calloc(1, strlen(plugins));
```

Comment:
I think this should do `calloc(1, strlen(plugins) + 1)` to account for the terminating NULL byte